wiki/Music_theory/Tempo/en.md

```diff
@@ -1,13 +1,34 @@
 ---
-stub: true
 tags:
   - BPM
+  - tempo
+  - beats per minute
 ---
 
 # Tempo
 
-![Screenshot of the BPM indicator on the beatmap information page](img/BPM_beatmap_info.png "The tempo can be seen on the top-corner of the beatmap information page.")
+*See also: [Timing](/wiki/Beatmapping/Timing)*
 
-**Tempo** is the speed at which a piece of music is played and is communicated, measured in [beats](/wiki/Music_theory/Beat) per minute (BPM).
+::: Infobox
+![Screenshot of the BPM indicator on the beatmap information page](img/beatmap-info.png "The song's tempo, equal to 187, can be seen on the top corner of the beatmap information page")
+:::
 
-The BPM of a song in a beatmap can be found on its information page next to the metronome symbol.
+**Tempo** refers to the speed of a song. It is typically measured in **beats per minute** (***BPM***), meaning the number of [whole musical beats](/wiki/Music_theory/Beat) in a minute of a song. For example, a tempo of 60 beats per minute signifies one beat per second, while a tempo of 120 beats per minute is twice as rapid, signifying two beats every second. The tempo directly affects a lot of gameplay aspects, such as the length of a single beat, [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity), or [kiai time](/wiki/Gameplay/Kiai_time). In osu! ecosystem, it is usually associated with a symbol of a [metronome](https://en.wikipedia.org/wiki/Metronome), the device used by musicians practise playing to a regular pulse.
+
+## Gameplay
+
+Many osu! elements base their timings around the current song's tempo:
+
+- Length of a [1/1 note](/wiki/Music_theory/Beat), which controls [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor) and — indirectly — the beatmap structure
```

Comment:
this feels like a weird thing to mention, i'd expect something along the lines of "gameplay becomes faster" instead